packages/zenn-cli/src/server/lib/helper.ts

```diff
@@ -163,6 +163,7 @@ export function generateFileIfNotExist(fullpath: string, content: string) {
 }
 
 export function completeHtml(html: string): string {
+  const errorStyle = `class="zenn-server-error-block" style="color: var(--c-error); font-weight: 700"`
```

Comment:
```suggestion
  const errorStyle = `data-body-error style="color: var(--c-error); font-weight: 700"`
```

このプロジェクトでは、classは基本的にスタイリングに用いたいと考えているので、今回の目的には `data-*` 属性を使用するようにしたいです。